MWT/how-to-background.rst

```diff
@@ -0,0 +1,43 @@
+How-To Define a Background
+==========================
+
+To create a custom background, a new class should be created, extending the ``Background`` class.
+Background subclasses have to implement two methods, as explained in the following sections.
+
+Informing whether the background is transparent
+-----------------------------------------------
+
+The ``isTransparent()`` method is called by the MWT framework in order to know whether or not the background is transparent.
+A background is considered as transparent if it does not draw every pixel with maximal opacity when it is applied.
+
+For example, the following snippet informs that the background is completely opaque:
+
+.. code-block:: Java
+
+	@Override
+	public boolean isTransparent() {
+		return false;
+	}
+
+Applying the background on a graphics context
+---------------------------------------------
+
+The ``apply(GraphicsContext g, Size size)`` method is called by the MWT framework in order to render the background and to set the background color of subsequent drawings.
```

Comment:
Indicate that setting the background color is not mandatory.

Reply:
then "set or remove"? i think doing either of them is mandatory

Reply:
Yes, good point!

Reply:
done